diff --git a/node_modules/r3f-perf/dist/components/TextsHighHZ.mjs b/node_modules/r3f-perf/dist/components/TextsHighHZ.mjs
index ca1e1ce..7abcbe5 100644
--- a/node_modules/r3f-perf/dist/components/TextsHighHZ.mjs
+++ b/node_modules/r3f-perf/dist/components/TextsHighHZ.mjs
@@ -7,7 +7,7 @@ import "../index.mjs";
 import { colorsGraph } from "./Perf.mjs";
 import * as THREE from "three";
 import { useEvent } from "@utsubo/events";
-import localFont from "../roboto.woff.mjs";
+/*import localFont from "../roboto.woff.mjs";*/
 import { getPerf } from "../store.mjs";
 const TextHighHZ = memo(
   ({
